src/app/components/crosssection/crosssection.component.ts

```diff
@@ -315,13 +317,10 @@ export class CrosssectionComponent implements AfterViewInit {
     track.addChild(image);
   }
   private addPanelHeader(widget: MultiWellWidget, track: CorrelationTrack, start: number, end: number) {
-    const headerGroup = widget.getTrackHeader(track);
+    const headerGroup = widget.getTrackHeader(track) as Group;
```

Comment:
try to fix in toolkit

Reply:
I could't set the exact type for this method in toolkit, seems there are too more dependencies